### ISSUE TITLE
Sidebar v2 working with crosstalk

### ISSUE THREAD

**Issue by ccamara** [trafficonese/leaflet.extras2#41]
Apparently, sidebar v2 is only working with shiny (https://github.com/trafficonese/leaflet.extras2/blob/master/inst/examples/sidebar_app.R), but I am wondering if it would be possible to make it work with crosstalk (https://rstudio.github.io/crosstalk/), so it could be deployed without a need for a shiny server.

**Comment by trafficonese** [comment]
Hey @ccamara , unfortunately I havent worked too much with crosstalk so my knowledge about it is very limited, but isn't it supposed to connect htmlwidgets together?
I am not sure how this plugin should be included with crosstalk, since the sidebar is just a placeholder for different inputs or outputs.
Do you mean markdown maybe?

**Comment by ccamara** [comment]
Good question, @trafficonese . 
Yes, crosstalk is aimed at implementing cross-widget interactions (currently, linked brushing and filtering). I do not know how it relates to sidebar, but I was asking because reading from the documentation it seemed that it requires shiny for some kind of reactiveness (not sure if I understand why, though).

So, let me rephrase my question: is it possible to use sidebar v2 in rmarkdown (without shiny)? If so, how?

**Comment by trafficonese** [comment]
Apparently it is possible :)

I just pushed [f0e009d](https://github.com/trafficonese/leaflet.extras2/commit/f0e009d99d14160c1c9fa2b1a128b9699694fbec) which gets the correct map-id, so the sidebar is appended correctly.

Here is a little markdown with a leaflet sidebar:
````
---
title: "Rmarkdown HTML including Leaflet Sidebar"
output: html_document
---

Show "Leaflet Sidebar" within Rmarkdown.

```{r echo=F, include=TRUE}
library(shiny)
library(leaflet)
library(leaflet.extras2)
sidebar_tabs(id = "mysidebarid",
  list(icon("car"), icon("user")),
  sidebar_pane(
    title = "home", id = "home_id", icon = icon("home"),
    sliderInput("obs", "Number of observations:",
                min = 1, max = 32, value = 10)
  ),
  sidebar_pane(
    title = "profile", id = "profile_id", icon = icon("wrench"),
      textInput("caption", "Caption", "Data Summary")
  )
)
```
```{r}
leaflet() %>%
  addTiles() %>%
  setView(lng = 5.0, lat = 51.0, zoom = 6) %>%
  addSidebar(id = "mysidebarid",
             options = list(position = "left"))
```

````

**Comment by ccamara** [comment]
Great, thanks! I will be playing with this to see if I can replace tooltips with information within the sidebar!
